src/install-dependencies.sh

```diff
@@ -18,8 +18,11 @@ ZOPFLIPNG_EXE="$ZOPFLIPNG_BUILD_DIR/zopflipng"
 PNGQUANT_VERSION_TAG="2.12.5"
 ZOPFLIPNG_VERSION_TAG="v2.2.0"
 LIBPNG_VERSION="1.6.37"
-LIBPNG_VERSION_DOWNLOAD="libpng16/$LIBPNG_VERSION/libpng-$LIBPNG_VERSION.tar.xz"
-LITTLECMS_VERSION="2.9"
+LIBPNG_VERSION_FILE="v$LIBPNG_VERSION.tar.gz"
+
+LITTLECMS_VERSION="2.12"
```

Comment:
Note to self: 

This PR updates the LittleCMS version to 2.12 from 2.9

Reply:
Current release is 2.13.1

https://github.com/mm2/Little-CMS/releases/tag/lcms2.13.1